client/src/components/CommentApp/components/Comment/index.tsx

```diff
@@ -629,6 +629,11 @@ export default class CommentComponent extends React.Component<CommentProps> {
                 }),
               );
             },
+            // Allow delay for side panel to open and comment card to fade in with animations
+            checkCanFocusTrap: () =>
+              new Promise((resolve) => {
+                setTimeout(resolve, 250);
+              }),
```

Comment:
I think we should first check if the element's available, just in case we can avoid the delay.

I have not run this code, but here's a rough start.

```suggestion
            // Allow delay for side panel to open and comment card to fade in with animations
            checkCanFocusTrap: ({ containers = [] }) => {
              if (containers.some((container) => container.querySelector('[data-focus-target="true"]'))) {
                return Promise.resolve();
              }
              return new Promise((resolve) => {
                setTimeout(resolve, 250);
              })
            },
```


> checkCanFocusTrap {(containers: Array<HTMLElement | SVGElement>) => Promise<void>}: Animated dialogs have a small delay between when onActivate is called and when the focus trap is focusable. checkCanFocusTrap expects a promise to be returned. When that promise settles (resolves or rejects), focus will be sent to the first tabbable node (in tab order) in the focus trap (or the node configured in the initialFocus option).

Reply:
@lb- I have updated the code to avoid delay for elements having focus target. I just wanted to confirm do I need to write tests for these changes?

Reply:
This area of code doesn't appear to have any unit tests sadly, for now you don't need to add tests for this as it would be a lot of work for this PR. 

Maybe a task for a future PR, even just creating a baseline test.